src/fc/SensorTask.cpp

```diff
@@ -90,6 +89,8 @@ void SensorTask::activity()
         sys.tasks.logger.log(str);
     }
 
+    sys.sensors.imu2.accel->setRange(BMI088Accel::RANGE_12G);
```

Comment:
Should add a comment here explaining why. Also idk what the right way to handle this kind of mission-specific config. Buried in this file seems sub-optimal, but it's fine for now.

Reply:
Oh yeah, that should be removed. Totally forgot about that lol